src/Uno.UWP/UI/Composition/CoreAnimation.iOSmacOS.cs

```diff
@@ -192,7 +194,7 @@ private void StopAnimation(StopReason reason, long? time = default, float? value
 
 		private EventHandler OnAnimationStarted(CAAnimation animation)
 		{
-			EventHandler handler = default;
+			EventHandler? handler = default;
```

Comment:
Codacy found an issue: [Remove this useless assignment to local variable 'handler'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5932478)